lib/dynamic_type/test/utils.h

```diff
@@ -51,5 +51,3 @@ struct DoubleInt64BoolVecTwo {
 };
 using IntSomeType = DynamicType<NoContainers, int, SomeType, NonInstantiable>;
 using BoolSomeType = DynamicType<NoContainers, bool, SomeType, NonInstantiable>;
-using SomeTypes =
```

Comment:
Now we are assuming that types in the DynamicType only appears once, I believe this is a fair assumption.